core/src/avm2/globals/flash/media/video.rs

```diff
@@ -1,5 +1,42 @@
 use crate::avm2::{Activation, Error, Object, TObject, Value};
-use crate::display_object::TDisplayObject;
+use crate::display_object::{TDisplayObject, Video};
+
+/// Implements `flash.media.Video`'s `init` method, which is called from the constructor
+pub fn init<'gc>(
+    activation: &mut Activation<'_, 'gc>,
+    this: Option<Object<'gc>>,
+    args: &[Value<'gc>],
+) -> Result<Value<'gc>, Error<'gc>> {
+    if let Some(this) = this {
+        activation.super_init(this, &[])?;
```

Comment:
I don't think this line is necessary, since the AS constructor will call `super` anyway.